LondonBritishLibrary/add/BLadd24989.xml

```diff
@@ -0,0 +1,206 @@
+                            <locus from="1r"/>
+                            <title type="complete" ref="LIT2598Yosipp"></title>
+                            <textLang mainLang="gez"/>
+                        <msItem xml:id="ms_i1.1">
+                            <locus from="1r"/>
+                            <title type="complete" ref="LIT2598Yosipp#Kefl1"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i1.2">
+                           <locus from="15v"/>
+                            <title type="complete" ref="LIT2598Yosipp#Kefl2"></title>
+                            <textLang mainLang="gez"></textLang>
+                        </msItem>
+                        <msItem xml:id="ms_i1.3">
+                            <locus from="31r"/>
+                            <title type="complete" ref="LIT2598Yosipp#Kefl3"></title>
+                            <textLang mainLang="gez"></textLang>
+                        </msItem>
+                        <msItem xml:id="ms_i1.4">
+                            <locus from="45v"/>
+                            <title type="complete" ref="LIT2598Yosipp#Kefl4"></title>
+                            <textLang mainLang="gez"></textLang>
+                        </msItem>
+                        <msItem xml:id="ms_i1.5">
+                            <locus from="60r"/>
+                            <title type="complete" ref="LIT2598Yosipp#Kefl5"></title>
+                            <textLang mainLang="gez"></textLang>
+                        </msItem>
+                        <msItem xml:id="ms_i1.6">
+                            <locus from="63v"/>
+                            <title type="complete" ref="LIT2598Yosipp#Kefl6"></title>
+                            <textLang mainLang="gez"></textLang>
+                        </msItem>
+                        <msItem xml:id="ms_i1.7">
+                            <locus from="73v"/>
+                            <title type="complete" ref="LIT2598Yosipp#Kefl7"></title>
+                            <textLang mainLang="gez"></textLang>
+                        </msItem>
+                        <msItem xml:id="ms_i1.8">
+                            <locus from="87v"/>
+                            <title type="complete" ref="LIT2598Yosipp#Kefl8"></title>
+                            <textLang mainLang="gez"></textLang>
+                        </msItem>
+                        <colophon xml:id="coloph1" xml:lang="gez">
+                          <locus target="#99r"></locus>
+                          ተፈጸመ፡ ጽሒፈ፡ መጽሐፈ፡ ዜና፡ አይሁድ፡ ዘአበይት፡ ኍልቍ፡ አክፋላቲሁ፡
+                          <sic resp="PRS10171Wright">፷</sic> ወኁልቍ፡
+                          ታሪኩ፡ <sic resp="PRS10171Wright">፷ አመ፡ ፯ ለሐምሴ፡</sic>
+                          ዕለተ፡ ዓርብ፡ በዘመነ፡ ዮሐንስ፡ ወንጌላዊ፡
+                          <date calendar="world" when="7353">በ፸፻ወ፫፻ወ፶ወ፫ ዓመተ፡ ዓለም።</date>
```

Comment:
Is this the 1861 given in origDate? If so, maybe you could add the converted date also here in the markup, that could be nice for readers.